ffi/diplomat/src/properties_maps.rs

```diff
@@ -6,41 +6,151 @@
+            provider: &ICU4XDataProvider,
+        ) -> DiplomatResult<Box<ICU4XCodePointMapData8>, ICU4XError> {
+            Self::try_get_prop_inner(provider, |p| maps::load_general_category(p))
+        }
+
+        /// Gets a map for Unicode property Bidi_Class from a [`ICU4XDataProvider`].
+        #[diplomat::rust_link(icu::properties::maps::load_bidi_class, Fn)]
+        pub fn try_get_bidi_class(
+            provider: &ICU4XDataProvider,
+        ) -> DiplomatResult<Box<ICU4XCodePointMapData8>, ICU4XError> {
+            Self::try_get_prop_inner(provider, |p| maps::load_bidi_class(p))
+        }
+
+        /// Gets a map for Unicode property East_Asian_Width from a [`ICU4XDataProvider`].
+        #[diplomat::rust_link(icu::properties::maps::load_east_asian_width, Fn)]
+        pub fn try_get_east_asian_width(
+            provider: &ICU4XDataProvider,
+        ) -> DiplomatResult<Box<ICU4XCodePointMapData8>, ICU4XError> {
+            Self::try_get_prop_inner(provider, |p| maps::load_east_asian_width(p))
+        }
+
+        /// Gets a map for Unicode property Line_Break from a [`ICU4XDataProvider`].
+        #[diplomat::rust_link(icu::properties::maps::load_line_break, Fn)]
+        pub fn try_get_line_break(
+            provider: &ICU4XDataProvider,
+        ) -> DiplomatResult<Box<ICU4XCodePointMapData8>, ICU4XError> {
+            Self::try_get_prop_inner(provider, |p| maps::load_line_break(p))
+        }
+
+        /// Gets a map for Unicode property Grapheme_Cluster_Break from a [`ICU4XDataProvider`].
+        #[diplomat::rust_link(icu::properties::maps::load_grapheme_cluster_break, Fn)]
+        pub fn try_grapheme_cluster_break(
+            provider: &ICU4XDataProvider,
+        ) -> DiplomatResult<Box<ICU4XCodePointMapData8>, ICU4XError> {
+            Self::try_get_prop_inner(provider, |p| maps::load_grapheme_cluster_break(p))
+        }
+
+        /// Gets a map for Unicode property Word_Break from a [`ICU4XDataProvider`].
+        #[diplomat::rust_link(icu::properties::maps::load_word_break, Fn)]
+        pub fn try_get_word_break(
+            provider: &ICU4XDataProvider,
+        ) -> DiplomatResult<Box<ICU4XCodePointMapData8>, ICU4XError> {
+            Self::try_get_prop_inner(provider, |p| maps::load_word_break(p))
+        }
+
+        /// Gets a map for Unicode property Sentence_Break from a [`ICU4XDataProvider`].
+        #[diplomat::rust_link(icu::properties::maps::load_sentence_break, Fn)]
+        pub fn try_get_sentence_break(
+            provider: &ICU4XDataProvider,
+        ) -> DiplomatResult<Box<ICU4XCodePointMapData8>, ICU4XError> {
+            Self::try_get_prop_inner(provider, |p| maps::load_sentence_break(p))
+        }
+
+        /// Gets a map for Unicode property Canonical_Combining_Class from a [`ICU4XDataProvider`].
+        #[diplomat::rust_link(icu::properties::maps::load_canonical_combining_class, Fn)]
+        pub fn try_get_canonical_combining_class(
```

Comment:
Discussion: We agreed last week to put CCC behind the experimental feature. Do we still want to do that, and how does that affect FFI?